apps/web/src/routes/AuthLayoutRouter/components/GlobalNav/GlobalNav.jsx

```diff
@@ -200,3 +198,5 @@ export default function GlobalNav (props) {
     </div>
   )
 }
+
+export { GlobalNavTooltipContainer }
```

Comment:
why are you exporting this here? the whole point is you dont want to export more than one thing from a component file. you should be able to just import this wherever it is used directly from the file